book/translation.rst

```diff
@@ -82,10 +84,10 @@ enable the ``translator`` in your configuration:
 
         // app/config/config.php
         $container->loadFromExtension('framework', array(
-            'translator' => array('fallback' => 'en'),
+            'translator' => array('fallbacks' => 'en'),
```

Comment:
Should we use `array('en')` here instead to indicate that you can provide more than one fallback?